### DISCUSSION TITLE
Support for point light sources?

### DISCUSSION THREAD

**Discussion by VHSDIR** [KilledByAPixel/LittleJS#77] [Q&A]
I am reviewing the game engine, studying examples and it looks cool and it seems to fit my needs for my desired project.

There is one gotcha: I will need to make levels mostly dark and add point source light to them.

As I searched the source code there is no support for lights - are there any plans to do lights and is it technically possible?

Great Job BTW 👏

**Comment by KilledByAPixel** [comment]
Thank you, that would be really cool! I have considered adding a component like that. 

Check out this article I wrote about 7 years ago that describes the system I used in another C++ game engine I made a long time ago and I think I could do something similar with WebGL.

https://frankforce.com/2d-light-mapping/
